mmcv/visualization/image.py

```diff
@@ -81,19 +85,19 @@ def imshow_bboxes(img,
     return img
 
 
-def imshow_det_bboxes(img,
-                      bboxes,
-                      labels,
-                      class_names=None,
-                      score_thr=0,
-                      bbox_color='green',
-                      text_color='green',
-                      thickness=1,
-                      font_scale=0.5,
-                      show=True,
-                      win_name='',
-                      wait_time=0,
-                      out_file=None):
+def imshow_det_bboxes(img: Union[str, np.ndarray],
+                      bboxes: np.ndarray,
+                      labels: np.ndarray,
+                      class_names: List[str] = None,
+                      score_thr: float = 0,
+                      bbox_color: Union[str, tuple] = 'green',
+                      text_color: Union[str, tuple] = 'green',
```

Comment:
```suggestion
                      text_color: Union[str, tuple, int, np.ndarray] = 'green',
```

Reply:
Please help update the type in https://github.com/open-mmlab/mmcv/pull/1946/files#diff-7ab945ee4fc4c1df5be4905c547ba7fb381e381a04876b1968444b4689c2df9bR111